sentinel/pkg/checker/balance/app.go

```diff
@@ -249,15 +254,30 @@ func loadWalletFromOraklAdmin(ctx context.Context, url string) ([]Wallet, error)
 
 func loadWalletFromPor(ctx context.Context, url string) (Wallet, error) {
 	wallet := Wallet{}
-	reporter, err := request.GetRequest[string](url+porEndpoint, nil, nil)
+	resp, err := request.UrlRequestRaw(url+porEndpoint, "GET", nil, nil, "")
 	if err != nil {
 		return wallet, err
 	}
-	address := common.HexToAddress(reporter)
+
+	if resp.StatusCode != http.StatusOK {
+		log.Info().
+			Int("status", resp.StatusCode).
+			Str("url", url+porEndpoint).
+			Msg("failed to make request")
```

Comment:
just a minor suggestion, how about more detailed error log? such as the file location, which function, etc. Or will that kind of info automatically show up in the logs?

Reply:
good point, will make an update